### PR TITLE
remove credits and update flow to match figma

### PR DESCRIPTION
**Pull request by jason-tung**
### Summary 

Updated user flow to exclude credits from onboarding.

https://www.notion.so/courseplan/Gray-out-Transfer-Equivalent-Cornell-Courses-from-Onboarding-9f0a1d34ea7041c0982e47ff5a82feb9
https://www.notion.so/courseplan/Remove-Credit-Column-from-Onboarding-219d43f739b64faea29154e55cfb0792

### Test Plan 

Onboarding should function as expected with the only exceptions being no credits column on transfer and review as well as the message the bottom of transfer.
image
image

### CODE REVIEW

**Review by SamChou19815 — CHANGES_REQUESTED** [review]
The implementation doesn't look like the design:
Screen Shot 2021-03-07 at 16 21 32

I'm also not a fan of commenting out code. We have version control so it's much better to remove them and then refer to this PR to add them back when we want it. Leaving out commented code increases maintenance burden, since they can't be type-checked and can become out of data with the surrounding code very easily, and makes people wonder their importance when trying to refactor the code.

File: src/components/Modals/Onboarding/Onboarding.scss [hunk 1]
```diff
@@ -245,6 +250,10 @@ input {
         margin-bottom: 2rem;
       }
     }
+
+    &--description {
+      margin-bottom: 1rem !important;
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/Onboarding.scss:255]: `!important` is not really best practice. can you think of an alternative solution?

File: src/components/Modals/Onboarding/OnboardingReview.vue [hunk 2]
```diff
@@ -116,6 +119,8 @@
                 }}</label>
               </div>
             </div>
+            <!-- 
+              TODO: BRING THIS BACK POSTLAUNCH
             <div class="alignCenter">
               <label class="onboarding-label">Credit</label>
               <div v-for="(exam, index) in onboardingData.exam" :key="'APCredit' + index">
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/OnboardingReview.vue:126]: Discussed with @einc and we agree with @SamChou19815 that deleting the code instead of commenting it out would prob be better & cleaner. Sorry about that! Can you delete the commented out code in your PR?

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Thanks Jason! Onboarding is coming together 😊. Requested some changes: main ones being deleting the commented out code and making sure that we don't have design discrepancies. Besides the frontend fixes that I mentioned in my comment for #347, there are some discrepancies in the Transfer page of Onboarding. Can you look through the FINAL FINAL FINAL SPEC for onboarding on Figma and check that the implementation matches the designs?

File: src/components/Modals/Onboarding/Onboarding.vue [hunk 3]
```diff
@@ -20,16 +20,13 @@
...
               />
-              Hi {{ name.firstName }}</span
+              Welcome to CoursePlan</span
             >
           </div>
-          <div v-if="!isEditingProfile" class="onboarding-description">
-            Let's get to know you first!
-          </div>
-          <div v-if="isEditingProfile" class="onboarding-description">Let's edit your profile!</div>
+          <div class="onboarding-description">Let's get to know you first</div>
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/Onboarding.vue:29]: Revert these changes since the onboarding modal should appear differently if a user is editing their profile and have gone through onboarding already vs. a user who is being onboarded for the first time.

File: src/components/Modals/Onboarding/OnboardingBasic.vue [hunk 4]
```diff
@@ -99,7 +99,8 @@ export default Vue.extend({
     const majorAcronyms = [...this.onboardingData.major];
     const minorAcronyms = [...this.onboardingData.minor];
     if (majorAcronyms.length === 0) majorAcronyms.push('');
-    if (minorAcronyms.length === 0) minorAcronyms.push('');
+    if (minorAcronyms.length === 0) minorAcronyms.push('', '');
+    if (minorAcronyms.length === 1) minorAcronyms.push('');
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/OnboardingBasic.vue:103]: What does this do?

File: src/components/Modals/Onboarding/OnboardingReview.vue [hunk 5]
```diff
@@ -89,13 +89,16 @@
...
+              <label class="onboarding-label">Subject</label>
               <div v-for="(exam, index) in onboardingData.exam" :key="'AP' + index">
                 <label v-if="exam.type == 'AP'" class="onboarding-label--review">{{
                   exam.subject
                 }}</label>
               </div>
-              <label class="onboarding-label addSpaceTop">IB Credits</label>
+              <label class="onboarding-label addSpaceTop onboarding-label--header"
+                >IB Credits</label
+              >
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/OnboardingReview.vue:101]: I think IB Credits should also have a label called Subject like AP Credits in the review page of Onboarding

File: src/components/Modals/Onboarding/OnboardingTransfer.vue [hunk 6]
```diff
@@ -225,7 +162,7 @@ export const getExamCredit = (exam: FirestoreAPIBExam): number => {
 
 export default Vue.extend({
   components: {
-    CourseSelector,
+    // CourseSelector,
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/OnboardingTransfer.vue:165]: If we don't use CourseSelector then I would just remove this instead of commenting it out.

**Review by tcho6319 — CHANGES_REQUESTED** [review]
Thanks Jason! Left some design discrepancies/issues:

1. I think instead of "College" and "Minor", it should be "Your Major" and "Your Minor". 
PR:
Screen Shot 2021-03-22 at 3 55 29 PM

Design:
Screen Shot 2021-03-22 at 3 56 14 PM

2. If this is not my first time going through Onboarding, it shouldn't say "Welcome..." but "Hi Theresa..."

PR:
Screen Shot 2021-03-22 at 3 57 47 PM

Master (intended behavior):
Screen Shot 2021-03-22 at 3 58 33 PM

3. Are we able to decrease the space between College and Major? This is more of a nit.
PR:
Screen Shot 2021-03-22 at 3 59 25 PM
 
Designs:
Screen Shot 2021-03-22 at 4 01 58 PM

4. The buttons to add another major and add another minor should be "+ another major" and "+ another minor". Similarly for "+ another subject" on the Transfer Credit page of Onboarding.
Screen Shot 2021-03-22 at 4 02 53 PM

Designs: Screen Shot 2021-03-22 at 4 03 38 PM

5. Can you remove the extra padding? Look over every Onboarding page to make sure it's removed.
Screen Shot 2021-03-22 at 4 05 18 PM

6. The dropdowns throughout the Onboarding modal seem a little off-center.
Screen Shot 2021-03-22 at 4 07 11 PM

7. You're missing the gray dropdown arrow
PR:
Screen Shot 2021-03-22 at 4 10 05 PM

Design (note that the dropdown for score should be same as subject so just one triangle):
Screen Shot 2021-03-22 at 4 10 18 PM

8. Can you add some space between the exam type and "Subject"?
PR:
Screen Shot 2021-03-22 at 4 18 23 PM

Designs:
Screen Shot 2021-03-22 at 4 19 11 PM

File: src/components/Modals/Onboarding/OnboardingBasic.vue (hunk 4 above)
Comment (jason-tung) [reply src/components/Modals/Onboarding/OnboardingBasic.vue:103]: image
wasnt sure if designs wanted two minor fields by default

File: src/components/Modals/Onboarding/OnboardingTransfer.vue (hunk 6 above)
Comment (jason-tung) [reply src/components/Modals/Onboarding/OnboardingTransfer.vue:165]: missed this when removing comments

File: src/components/Modals/Onboarding/OnboardingReview.vue (hunk 5 above)
Comment (jason-tung) [reply src/components/Modals/Onboarding/OnboardingReview.vue:101]: I don't follow

File: src/components/Modals/Onboarding/OnboardingBasic.vue (hunk 4 above)
Comment (einc) [reply src/components/Modals/Onboarding/OnboardingBasic.vue:103]: oh sorry this looks like a mistake. Should just be one

File: src/components/Modals/Onboarding/OnboardingTransfer.vue (hunk 6 above)
Comment (einc) [reply src/components/Modals/Onboarding/OnboardingTransfer.vue:165]: Screen Shot 2021-03-26 at 11 14 35 AM

File: src/components/Modals/Onboarding/OnboardingReview.vue (hunk 5 above)
Comment (tcho6319) [reply src/components/Modals/Onboarding/OnboardingReview.vue:101]: There is no Subject for IB Credits as seen below:
Screen Shot 2021-03-26 at 11 46 02 AM

File: src/components/Modals/Onboarding/Onboarding.scss [hunk 7]
```diff
@@ -432,6 +451,22 @@ input {
         margin-right: 8.7px;
         margin-left: 5px;
       }
+
+      &.college-test-credits-arrow {
+        border-left: 6.24px solid transparent;
+        border-right: 6.24px solid transparent;
+
+        border-top: 6.24px solid #c4c4c4;
```
Comment (tcho6319) [inline src/components/Modals/Onboarding/Onboarding.scss:459]: Nit: I believe `#c4c4c4` is `$inactiveGray`

**Review by tcho6319 — APPROVED** [review]
Overall LGTM. Thanks Jason! Left a couple nits (including the one below). If you could address them, that would be awesome. 

1. Should say "+ another subject" to be consistent with how we add another major/minors
Screen Shot 2021-03-28 at 6 32 02 PM

**Review by SamChou19815 — APPROVED** [review]
